dataplane/linux/vxlan_mgr.go

```diff
@@ -34,12 +35,26 @@ import (
 	"github.com/projectcalico/felix/routetable"
 )
 
+// added so that we can shim netlink for tests
+type vxlanDataplane interface {
```

Comment:
None of these seem like VXLAN specific functions - I might call this interface something to do with netlink, since that is what it seems to be wrapping. e.g., `type netlinkHandle interface`

Reply:
I just did this because there's another one for the IPIP stuff. I'll rename this though